ftplugin/fish.vim

```diff
@@ -7,6 +7,12 @@ setlocal formatoptions-=t
 setlocal include=\\v^\\s*\\.>
 setlocal iskeyword=@,48-57,-,_,.,/
 setlocal suffixesadd^=.fish
+" show existing tab with 4 spaces width
```

Comment:
I'm not sure addons should dictate the indentation style for filetypes, unless necessary for that filetype (like with makefiles that require tabs). So `tabstop shiftwidth expandtab` probably shouldn't be set here; they should be set by the user and/or in a modeline.